proto/types/types.proto

```diff
@@ -169,3 +170,16 @@ message SignedHeader {
   Header header = 1;
   Commit commit = 2;
 }
+
+message ValidatorSet {
+  repeated Validator validators         = 1;
+  Validator          proposer           = 2;
+  uint64             total_voting_power = 3;
+}
+
+message Validator {
+  bytes                             address           = 1;
```

Comment:
Does Tendermint use YAML at all for human-readability/`String()` implementations? If so, we should add YAML tags.

Reply:
TM doesn't use YAML.